shared/chat/routes.js

```diff
@@ -69,19 +69,19 @@ const chatChildren = {
   },
   attachmentFullscreen: {
     component: AttachmentFullscreen,
-    tags: makeLeafTags(isMobile ? {hideStatusBar: true, fullscreen: true} : {layerOnTop: true}),
+    tags: makeLeafTags(
+      isMobile ? {hideStatusBar: true, fullscreen: true, underNotch: true} : {layerOnTop: true}
+    ),
     children: key => makeRouteDefNode(chatChildren[key]),
   },
   attachmentVideoFullscreen: {
```

Comment:
Is there any way to reach this on mobile?

Reply:
discuss w/ @mmaxim and we think not. i'll try and remove it

Reply:
Before the inline player was a thing, on Android it would show this, but I think that is gone now.

Reply:
i removed it